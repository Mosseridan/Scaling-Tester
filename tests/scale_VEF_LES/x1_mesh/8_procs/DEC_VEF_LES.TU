Statistiques de post resolution

Temps total                       5.08593

