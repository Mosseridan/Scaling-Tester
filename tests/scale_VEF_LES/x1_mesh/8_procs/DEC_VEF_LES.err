Petsc initialization succeeded.
Parallel engine initialized : Comm_Group_Noparallel with 1 processors
Executable: /home/idanmos/CEA/no_omp/TrioCFD/TrioCFD_opt
Data file name = DEC_VEF_LES
feenableexcept enabled.
Data file : DEC_VEF_LES
Localisation etude: /home/idanmos/projects/Scaling-Tester/tests/scale_VEF_LES/x1_mesh/8_procs
Nom du cas DEC_VEF_LES
 code : /home/idanmos/CEA/no_omp/TrioCFD/TrioCFD_opt
 version : 1.7.6 
Debut de l'execution 
MAIN: Reading and executing data file
Dimension::interpreter : dimension = 3
It treats a calculation with a precision 1e-05
It deals with a calculation using precision format 5
Execution of the interpreter Lire_Tgrid
Reading a mesh which comes from Tgrid
Association of the domain on which the interpreter applies Lire_Tgrid
Reading of the file Tube_coude_463259.msh ...
The total number of elements to read is 463259
3D elements of type Tetrahedron
Reading a comment:
(0

Reading of the dimension of the case:
Dimension 3.

Reading a comment:
(0

The total number of nodes to read is 89710

89710 nodes are read in the area 8


The type of area 9 is 1 (1:fluid, 17:solid)

The total number of faces to read is 950399

Reading a comment:
(0

902637 internal faces are read in the area 10

Reading a comment:
(0

19438 faces are read from the boundary number 11

Reading a comment:
(0

27656 faces are read from the boundary number 12

Reading a comment:
(0

334 faces are read from the boundary number 13

Reading a comment:
(0

334 faces are read from the boundary number 14

Reading a comment:
(0

Reading of a name:
The area 9 is called fluid

Reading of a name:
The area 10 is called interieur

Reading of a name:
The area 11 is called symmetry

Reading of a name:
The area 12 is called wall

Reading of a name:
The area 13 is called outlet

Reading of a name:
The area 14 is called inlet

Elimination of 0 unnecessary nodes in the domain dom
Execution of the interpreter Dilate
Association of the domain on which the interpreter applies Dilate
Expands the domain dom of a factor 0.001
Execution of the interpreter Transformer
Association of the domain on which the interpreter applies Transformer
Reading and interpretation of the function x
Interpretation of the function x Ok
Reading and interpretation of the function y
Interpretation of the function y Ok
Reading and interpretation of the function -z
Interpretation of the function -z Ok
Execution of the interpreter ExtrudeBord
Association of the domain on which the interpreter applies ExtrudeBord
number of 3D nodes :89710
number of 3D faces of inlet :334
number of nodes per faces in 3D :3
boundary maintained symmetry
boundary maintained wall
boundary to eliminate outlet
Decouper : Splitting of a domain
 Domain name to split : box
==============================================
The extreme coordinates of the domain box are:
x is between -0.5334 and -0.4572
y is between -0.1624 and -0.1524
z is between 0 and 0.0381
==============================================
==============================================
The volume cells of the domain box are:
sum(volume cells)= 2.27368e-05
mean(volume cells)= 1.21561e-09
min(volume cells)= 5.47064e-10
max(volume cells)= 4.81391e-09
==============================================
domaine = box
 Creation of a partitioner of type: Partitionneur_metis
Reading parameters of Partitionneur_Metis
Partitionneur_Metis::construire_partition
 Construction of graph connectivity...
 Construction of the som_elem connectivity
 Construction of the elem_elem connectivity
===============
Call for PMETIS
===============
===============
Partitioning quality : edgecut = 899
-> It is roughly the total number of edges (faces) which will be shared by the processors.
-> The lesser this number is, the lesser the total volume of communication between processors.
-> You can increase nb_essais option (default 1) to try to reduce (but at a higher CPU cost) this number.
===============
Correction elem0 on processor 0
Correction of the splitting to put the element 0 on processor 0.
 Exchange of parts 0 and 7
The partitioner has generated 8 parts.
Search of neighboring elements of boundary faces
Construction of lists of elements per subdomain
Generation of 8 parts:
 Construction of part number 0
  Number of nodes    : 608
  Number of elements : 2338
  Number of joints   : 2
  Joint 0 PeVoisin 5 NbSommets 34 NbFaces 50 NbElemDist 599
  Joint 1 PeVoisin 6 NbSommets 73 NbFaces 120 NbElemDist 1410
               Total:    NbSommets 107 NbFaces 170 NbElemDist 2009
Writing part 0 into the binary file box_0000.Zones
Writing of 608 nodes.
Writing of 2338 elements.
 Construction of part number 1
  Number of nodes    : 607
  Number of elements : 2338
  Number of joints   : 4
  Joint 0 PeVoisin 2 NbSommets 1 NbFaces 0 NbElemDist 90
  Joint 1 PeVoisin 3 NbSommets 56 NbFaces 89 NbElemDist 1136
  Joint 2 PeVoisin 4 NbSommets 54 NbFaces 86 NbElemDist 1056
  Joint 3 PeVoisin 7 NbSommets 37 NbFaces 54 NbElemDist 856
               Total:    NbSommets 148 NbFaces 229 NbElemDist 3138
Writing part 1 into the binary file box_0001.Zones
Writing of 607 nodes.
Writing of 2338 elements.
 Construction of part number 2
  Number of nodes    : 626
  Number of elements : 2338
  Number of joints   : 3
  Joint 0 PeVoisin 1 NbSommets 1 NbFaces 0 NbElemDist 65
  Joint 1 PeVoisin 3 NbSommets 68 NbFaces 110 NbElemDist 1447
  Joint 2 PeVoisin 7 NbSommets 42 NbFaces 61 NbElemDist 652
               Total:    NbSommets 111 NbFaces 171 NbElemDist 2164
Writing part 2 into the binary file box_0002.Zones
Writing of 626 nodes.
Writing of 2338 elements.
 Construction of part number 3
  Number of nodes    : 610
  Number of elements : 2338
  Number of joints   : 3
  Joint 0 PeVoisin 1 NbSommets 56 NbFaces 89 NbElemDist 1198
  Joint 1 PeVoisin 2 NbSommets 68 NbFaces 110 NbElemDist 1302
  Joint 2 PeVoisin 7 NbSommets 21 NbFaces 26 NbElemDist 557
               Total:    NbSommets 145 NbFaces 225 NbElemDist 3057
Writing part 3 into the binary file box_0003.Zones
Writing of 610 nodes.
Writing of 2338 elements.
 Construction of part number 4
  Number of nodes    : 616
  Number of elements : 2338
  Number of joints   : 4
  Joint 0 PeVoisin 1 NbSommets 54 NbFaces 86 NbElemDist 1226
  Joint 1 PeVoisin 5 NbSommets 70 NbFaces 113 NbElemDist 1397
  Joint 2 PeVoisin 6 NbSommets 6 NbFaces 4 NbElemDist 136
  Joint 3 PeVoisin 7 NbSommets 31 NbFaces 44 NbElemDist 649
               Total:    NbSommets 161 NbFaces 247 NbElemDist 3408
Writing part 4 into the binary file box_0004.Zones
Writing of 616 nodes.
Writing of 2338 elements.
 Construction of part number 5
  Number of nodes    : 610
  Number of elements : 2338
  Number of joints   : 4
  Joint 0 PeVoisin 0 NbSommets 34 NbFaces 50 NbElemDist 806
  Joint 1 PeVoisin 4 NbSommets 70 NbFaces 113 NbElemDist 1341
  Joint 2 PeVoisin 6 NbSommets 43 NbFaces 65 NbElemDist 963
  Joint 3 PeVoisin 7 NbSommets 9 NbFaces 8 NbElemDist 276
               Total:    NbSommets 156 NbFaces 236 NbElemDist 3386
Writing part 5 into the binary file box_0005.Zones
Writing of 610 nodes.
Writing of 2338 elements.
 Construction of part number 6
  Number of nodes    : 624
  Number of elements : 2338
  Number of joints   : 4
  Joint 0 PeVoisin 0 NbSommets 73 NbFaces 120 NbElemDist 1334
  Joint 1 PeVoisin 4 NbSommets 6 NbFaces 4 NbElemDist 340
  Joint 2 PeVoisin 5 NbSommets 43 NbFaces 65 NbElemDist 889
  Joint 3 PeVoisin 7 NbSommets 44 NbFaces 69 NbElemDist 907
               Total:    NbSommets 166 NbFaces 258 NbElemDist 3470
Writing part 6 into the binary file box_0006.Zones
Writing of 624 nodes.
Writing of 2338 elements.
 Construction of part number 7
  Number of nodes    : 614
  Number of elements : 2338
  Number of joints   : 6
  Joint 0 PeVoisin 1 NbSommets 37 NbFaces 54 NbElemDist 793
  Joint 1 PeVoisin 2 NbSommets 42 NbFaces 61 NbElemDist 1124
  Joint 2 PeVoisin 3 NbSommets 21 NbFaces 26 NbElemDist 692
  Joint 3 PeVoisin 4 NbSommets 31 NbFaces 44 NbElemDist 799
  Joint 4 PeVoisin 5 NbSommets 9 NbFaces 8 NbElemDist 492
  Joint 5 PeVoisin 6 NbSommets 44 NbFaces 69 NbElemDist 928
               Total:    NbSommets 184 NbFaces 262 NbElemDist 4828
Writing part 7 into the binary file box_0007.Zones
Writing of 614 nodes.
Writing of 2338 elements.
End of the interpreter Decouper
MAIN: End of data file
Arret des processes.
