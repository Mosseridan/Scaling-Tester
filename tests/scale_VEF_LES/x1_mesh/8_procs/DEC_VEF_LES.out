-------------------------------------------------------------------
 
                          TRUST
                      version : 1.7.6 
                          CEA - DEN
 

 0 file(s) modified 


 405 file(s) created 

 ------------------------------------------------------------------
 
  Vous traitez le cas DEC_VEF_LES
 
Reading data file 

13 MBytes of RAM taken by the calculation.
26 MBytes of RAM taken by the calculation.
26 MBytes of RAM taken by the calculation.
27 MBytes of RAM taken by the calculation.
 Runtime parameters:
   Objective type: METIS_OBJTYPE_CUT
   Coarsening type: METIS_CTYPE_SHEM
   Initial partitioning type: METIS_IPTYPE_GROW
   Refinement type: METIS_RTYPE_FM
   Perform a 2-hop matching: No
   Number of balancing constraints: 1
   Number of refinement iterations: 10
   Random number seed: -1
   Number of partitions: 8
   Number of cuts: 1
   User-supplied ufactor: 1
   Target partition weights: 
        0=[1.25e-01]   1=[1.25e-01]   2=[1.25e-01]   3=[1.25e-01]   4=[1.25e-01]
        5=[1.25e-01]   6=[1.25e-01]   7=[1.25e-01]
   Allowed maximum load imbalance: 1.001 

     18704      73476      73476 [20] [     1402:   18704 ]
     10391      51118      56850 [20] [     1402:   18704 ]
      5688      30712      45056 [20] [     1402:   18704 ]
      3110      16892      34408 [20] [     1402:   18704 ]
      1691       9218      24710 [20] [     1402:   18704 ]
       908       5870      17614 [20] [     1402:   18704 ]
       482       3792      13260 [20] [     1402:   18704 ]
       256       2336      10472 [20] [     1402:   18704 ]
       137       1364       8524 [20] [     1402:   18704 ]
        72        714       6842 [20] [     1402:   18704 ]
        37        332       5464 [20] [     1402:   18704 ]
        20        152       4284 [20] [     1402:   18704 ]
      9352      36518      36518 [20] [      701:    9352 ]
      5177      25296      28168 [20] [      701:    9352 ]
      2863      15366      22364 [20] [      701:    9352 ]
      1559       8322      17030 [20] [      701:    9352 ]
       848       4492      12190 [20] [      701:    9352 ]
       456       2846       8590 [20] [      701:    9352 ]
       246       1868       6398 [20] [      701:    9352 ]
       131       1144       5034 [20] [      701:    9352 ]
        69        624       4018 [20] [      701:    9352 ]
        35        312       3254 [20] [      701:    9352 ]
        19        142       2584 [20] [      701:    9352 ]
      4676      18052      18052 [20] [      350:    4676 ]
      2604      12558      13908 [20] [      350:    4676 ]
      1432       7404      10958 [20] [      350:    4676 ]
       784       4000       8222 [20] [      350:    4676 ]
       435       2164       5938 [20] [      350:    4676 ]
       240       1350       4134 [20] [      350:    4676 ]
       129        856       3076 [20] [      350:    4676 ]
        69        518       2434 [20] [      350:    4676 ]
        37        276       1912 [20] [      350:    4676 ]
        19        128       1442 [20] [      350:    4676 ]
      4676      18114      18114 [20] [      350:    4676 ]
      2599      12472      13960 [20] [      350:    4676 ]
      1418       7382      10940 [20] [      350:    4676 ]
       771       3918       8296 [20] [      350:    4676 ]
       425       2126       5884 [20] [      350:    4676 ]
       229       1334       4146 [20] [      350:    4676 ]
       122        852       3068 [20] [      350:    4676 ]
        66        504       2378 [20] [      350:    4676 ]
        35        282       1904 [20] [      350:    4676 ]
        19        142       1476 [20] [      350:    4676 ]
      9352      36544      36544 [20] [      701:    9352 ]
      5204      25312      28248 [20] [      701:    9352 ]
      2856      15224      22402 [20] [      701:    9352 ]
      1547       8392      16980 [20] [      701:    9352 ]
       852       4550      12224 [20] [      701:    9352 ]
       460       2862       8710 [20] [      701:    9352 ]
       245       1846       6542 [20] [      701:    9352 ]
       130       1112       5244 [20] [      701:    9352 ]
        68        628       4172 [20] [      701:    9352 ]
        36        322       3396 [20] [      701:    9352 ]
        19        148       2716 [20] [      701:    9352 ]
      4676      18160      18160 [20] [      350:    4676 ]
      2589      12452      13986 [20] [      350:    4676 ]
      1405       7326      10974 [20] [      350:    4676 ]
       765       3990       8292 [20] [      350:    4676 ]
       413       2122       5904 [20] [      350:    4676 ]
       222       1354       4190 [20] [      350:    4676 ]
       119        854       3104 [20] [      350:    4676 ]
        65        512       2442 [20] [      350:    4676 ]
        35        278       1976 [20] [      350:    4676 ]
        20        150       1622 [20] [      350:    4676 ]
      4676      18146      18146 [20] [      350:    4676 ]
      2584      12478      13962 [20] [      350:    4676 ]
      1419       7456      11018 [20] [      350:    4676 ]
       777       4086       8368 [20] [      350:    4676 ]
       427       2250       6022 [20] [      350:    4676 ]
       232       1418       4252 [20] [      350:    4676 ]
       122        902       3226 [20] [      350:    4676 ]
        64        536       2532 [20] [      350:    4676 ]
        33        280       1956 [20] [      350:    4676 ]
        18        142       1598 [20] [      350:    4676 ]

Timing Information -------------------------------------------------
 Multilevel: 		   0.058
     Coarsening: 		   0.038
            Matching: 			   0.020
            Contract: 			   0.015
     Initial Partition: 	   0.001
     Uncoarsening: 		   0.015
          Refinement: 			   0.008
          Projection: 			   0.007
     Splitting: 		   0.003
********************************************************************

 gk_mcore statistics
           coresize:       225000         nmops:         2048  cmop:      0
        num_callocs:          790   num_hallocs:            1
       size_callocs:      3899744  size_hallocs:        74816
        cur_callocs:            0   cur_hallocs:            0
        max_callocs:       224472   max_hallocs:        74816
 nbrpool statistics
        nbrpoolsize:            0   nbrpoolcpos:            0
    nbrpoolreallocs:            0

28 MBytes of RAM taken by the calculation.

--------------------------------------------
clock: Total execution: 5.08593 s
